packages/tools/devtools/devtools-browser-extension/src/BackgroundConnection.ts

```diff
@@ -13,19 +14,31 @@ import {
 	devtoolsMessageSource,
 } from "@fluidframework/devtools-core";
 
-import { browser } from "../Globals";
+import { browser } from "./Globals";
 import {
 	devToolsInitAcknowledgementType,
 	type DevToolsInitMessage,
 	devToolsInitMessageType,
 	extensionMessageSource,
-	postMessageToPort,
 	type TypedPortConnection,
-} from "../messaging";
-import {
-	devtoolsScriptMessageLoggingOptions,
-	formatDevtoolsScriptMessageForLogging,
-} from "./Logging";
+} from "./messaging";
+
+/**
+ * {@link BackgroundConnection} input parameters.
+ */
+export interface BackgroundConnectionParameters {
+	/**
+	 * All messages sent through the returned instance's {@link BackgroundConnection.postMessage}
+	 * method will get this value written to their 'source' property.
+	 * @see {@link @fluidframework/devtools-core#ISourcedDevtoolsMessage}
```

Comment:
Nit. Feels a bit clearer given the new location of the docs.
```suggestion
	 * This value will get written to the 'source' property of all messages sent with the {@link BackgroundConnection.postMessage} method of the {@link BackgroundConnection} instance that
	 * gets passed this props object.
	 * @see {@link @fluidframework/devtools-core#ISourcedDevtoolsMessage}
```

Reply:
Updated with some adjustments.